### PR TITLE
Tohirjon refactor

### PR DESCRIPTION
**Pull request by TohirjonIbnAlisher**
removed redundant comment.

### CODE REVIEW

File: Lesson01/LMS/MainWindow.xaml.cs [hunk 1]
```diff
@@ -23,7 +23,8 @@ public partial class MainWindow : Window
         public MainWindow()
         {
             InitializeComponent();
-// comment
+
+            int a = "hello";
```
Comment (Mirazyzz) [inline Lesson01/LMS/MainWindow.xaml.cs:27]: It is not possible to assign "hello" to integer type, please change it data type to string.

**Review by Mirazyzz — CHANGES_REQUESTED** [review]

**Review by Mirazyzz — APPROVED** [review]
LGTM!

File: Lesson01/LMS/MainWindow.xaml.cs [hunk 2]
```diff
@@ -23,7 +23,8 @@ public partial class MainWindow : Window
         public MainWindow()
         {
             InitializeComponent();
-// comment
+
+            string a = "hello";
```
Comment (Boburm1rzo) [inline Lesson01/LMS/MainWindow.xaml.cs:27]: ok

**Review by Boburm1rzo — APPROVED** [review]
you can merge

**Comment by TohirjonIbnAlisher** [comment]
Thanks!

**Pull request merged**